modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-disk-partitioning_{context}"]
+= Configuring disk partitioning with SiteConfig
+
+You can use the `SiteConfig` YAML file to generate the `MachineConfig` CR used for disk partitioning. You need to to modify values in the `SiteConfig` CR to reflect dependencies on the underlying disk.
+The values must be established prior to installation.
+
+[NOTE]
+====
+You must use persistent naming for devices to avoid device names such as `/dev/sda` and `/dev/sdb` being switched at every reboot. You can use `rootDeviceHints` to choose the bootable device and then use same device for further partitioning: in this case, for Image registry.
+====
+
+.Prerequisites
+
+
+* You have installed and configured Zero Touch Provisioning (ZTP). For further information on this, see the topic on ZTP in Additional resources.
+
+.Procedure
+
+. Add the following YAML to the `SiteConfig.yaml` file to generate `MachineConfig` for disk partitioning. :
++
+[source,yaml]
+----
+nodes:
+  - rootDeviceHints:
+      wwn: "0x62cea7f05c98c2002708a0a22ff480ea"
+    diskPartition:
+    - device: /dev/disk/by-id/wwn-0x62cea7f05c98c2002708a0a22ff480ea <1>
+      partitions:
+       - mount_point: /var/imageregistry
+         size: 102500 <2>
+         start: 344844 <3>
+----
+<1> This setting depends on the hardware. The setting can be a serial number or device name. The value must match the entry for `rootDeviceHint`.
+<2> The minimum value for image registry size is 100 gig (102500 MiB). For further information, see the Image Registry topic in Additional Resources.
+<3> The minimum value for the mount point is 25000 MiB. Otherwise the root partition is too small. Select `startMiB` (Disk - sizeMiB - some buffer). The total of startMiB + sizeMiB must not exceed the disk size, or the installation will fail.
```

Comment:
"Otherwise the root partition is too small" is an unnecessary sentence, given the previous sentence defines the minimum value. 

Is "(Disk - sizeMiB - some buffer)" a math equation like "startMiB + sizeMiB"? Please find another way to clearly state what is expected of the user.